### ISSUE TITLE
Self Recording: “Video trimming in progress/completed” is not announced to a screen reader

### ISSUE THREAD

**Issue by bmattb** [swissspidy/web-stories-wp#12137]
## Bug Description

When using self recording there is no announcement made to screen readers on completion of video trimming

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:
